mantidproject/mantid
# BioSANS sensitivity correction

`SensitivityCorrection` was not working for BioSANS (new wing detector)

Now, if `MaskComponent("component name")` is used, `SensitivityCorrection` discards this detector for the correction.

`Mask(nx_low=0, nx_high=0, ny_low=5, ny_high=5, component_name="component name")` was not masking all edges. It is now fixed.

See #16674

**To test:**

This branch is is analysis in `/HFIR/CG3/shared/mantid/Build`. You can test it from there using the script:
`/HFIR/CG3/shared/data/CG3_mantid_test/Exp318.py`
Commenting out the Sensitivity to see the result in Iq.

For the Mask, either play with `Mask` and number of edges masked, or use `SANSMask` and play with the MaskEdges list.

Fixes #16674.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
